### PR TITLE
Mentions `flutter packages get` rather than `pub get`

### LINKED ISSUE #6417: Update all tool string containing 'pub get' to 'flutter packages get'
We recently changed flutter so that you need to run `flutter packages get`, not `pub get`.

However, some (many?) status messages still say that we are running `pub get` thus potentially causing confusion. For example, when running flutter for the first time:

```
$ flutter doctor
Downloading Material fonts...                        2.2s
Downloading package sky_engine...                    1.4s
Running 'pub get' in sky_engine...                   0.4s
Downloading package sky_services...                  0.4s
Running 'pub get' in sky_services...                 3.1s
Downloading package flutter_services...              0.3s
Running 'pub get' in flutter_services...             1.7s
```

### PR DESCRIPTION
**Pull request by abarth**
Fixes #6417

### CODE REVIEW

**Comment by abarth** [comment]
@danrubel @Hixie

File: examples/flutter_gallery/flutter_gallery.iml [hunk 1]
```diff
@@ -3,8 +3,6 @@
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <excludeFolder url="file://$MODULE_DIR$/.pub" />
```
Comment (danrubel) [inline examples/flutter_gallery/flutter_gallery.iml]: I think `.pub` is an implementation detail for both `pub` and `flutter packages` and should be excluded.
@devoncarew might know for sure.

**Review by danrubel — APPROVED** [review]
LGTM with one question

File: examples/flutter_gallery/flutter_gallery.iml (hunk 1 above)
Comment (abarth) [inline examples/flutter_gallery/flutter_gallery.iml]: Oh sorry.  This is just intellij fighting itself.

**Pull request merged**